src/browser/NativeMessageInstaller.cpp

```diff
@@ -220,15 +228,17 @@ QString NativeMessageInstaller::getNativeMessagePath(SupportedBrowsers browser)
     // defined in the host, so we must hardcode them here.
     if (browser == SupportedBrowsers::TOR_BROWSER) {
         basePath = QDir::homePath() + "/.local/share";
-    } else if (browser == SupportedBrowsers::FIREFOX) {
+    } else if (browser == SupportedBrowsers::FIREFOX || browser == SupportedBrowsers::LIBREWOLF)
         basePath = QDir::homePath();
-    } else {
-        basePath = QDir::homePath() + "/.config";
-    }
+}
+else
+{
+    basePath = QDir::homePath() + "/.config";
+}
```

Comment:
I don't understand how this passes code formatting? This is clearly wrong indentation here, and inconsistent with the rest of the code...

Reply:
I wondered about that too. Before I ran the code formatter, it was better, I made sure to format everything correctly.